roles/shell/tasks/nvim.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Download nvim image
+  ansible.builtin.get_url:
+    url: https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
+    dest: "{{ '/home/' + user + '/nvim.appimage' if user != 'root' else '/root/nvim.appimage' }}"
+    mode: '0755'
+
+- name: Run nvim.appimage briefly in the background
+  command: "{{ '/home/' + user + '/nvim.appimage' if user != 'root' else '/root/nvim.appimage' }}"
+  async: 5
+  poll: 0
+
+- name: Ensure nvim.appimage is not running
+  command: pkill -f nvim.appimage
```

Comment:
command module are known as bad practice in Ansible, is there a way to implement it better?